src/content/articles/alab.md

```diff
@@ -88,30 +88,47 @@ downloadSection: {
   # The download links and button setup for the download table.
   downloads: [{
     buttonText: "DOWNLOAD",
-    downloadUrl: "https://dpel-assets.aswf.io/usd-alab/alab-v2.1.0.zip",
+    downloadUrl: "https://dpel-assets.aswf.io/usd-alab/alab-v2.2.0.zip",
     size: "9.3 GB",
-    description: "USD Scene - v.2.1.0",
+    description: "",
+    descriptionBold: "Asset Structure | ",
+    extraDescription: "Main asset structure for the ALab showcasing how all assets relate to each other through USD composition arcs. This is purely the USD structure linking all files, and does not include any geometry, shaders or lights.",
```

Comment:
I was looking into either creating a new column for "Description" on the existing table, or extending the content of one of the existing cells. I went for the later taking the intel one as an example:

https://github.com/AcademySoftwareFoundation/dpel-website/blob/c87e59661c73db9045c5cb1fe585fb6f75429937/src/content/articles/intel-cloud-library.md?plain=1#L83-L85

If there's a better way please let me know!

Reply:
I think sticking with the 3-column layout and not adding a 4th works fine 👍